embedingModels/README.md

```diff
@@ -0,0 +1,61 @@
+# Embeding-api
+
+## How run servis
+
+### GPU option
+
+1. Instal git lfs
+2. NVIDIA Docker Toolkit
+3. Add shh key to hugingface  account
+4. run this in ChatKNML/ :
+
+```sh
+cd embeding_models
+git clone git@hf.co:intfloat/e5-large-v2
+```
+
+### Aleternativ for 1,2,3
+
+Copy evry flie form here:
+
+#### IMPORTANT
+
+keep file tree like this !!!
+
+```sh
+embeding_models
+├── e5-large-v2
+│   ├── 1_Pooling
+│   │   └── config.json
+│   ├── config.json
+│   ├── handler.py
+│   ├── model.safetensors
+│   ├── modules.json
+│   ├── pytorch_model.bin
+│   ├── README.md
+│   ├── sentence_bert_config.json
+│   ├── special_tokens_map.json
+│   ├── tokenizer_config.json
+│   ├── tokenizer.json
+│   └── vocab.txt
+└── README.md
+```
```

Comment:
**suggestion:** just don't mess with submodule, as it's another repo you don't have permissions for :v

Reply:
I'll improve to download only models in the next week